queries/highlights.scm

```diff
@@ -1,17 +1,28 @@
-(ERROR) @error
+(comment) @comment
 
-(line_comment) @comment
+(definition) @keyword
 
 (identifier) @variable
 
+(label_name) @label
+
 (token) @string
 
 [
   "="
   "|"
+] @operator
+
+[
+  "*"
+  "?"
+] @repeat
```

Comment:
Well, I guess neovim differs here from different editors like helix, is this a "standardized" tag (i.e. used outside of neovim as well?)
Is it possible otherwise to give it multiple tags, like `@repeat` and `@operator.repeat`?

Reply:
Yeah, I believe `@clason` has mentioned interest in collaborating with other editors like emacs and helix to unify and define a base parser query definition. This is something that I think is reallly important, but no one seems to want to get it going (and who does? it'd be extremely hard getting everyone together at first to discuss this)

We can keep the definition for helix here separate as nvim-treesitter (for now, until potential parser unification) uses the queries defined in its queries folder, not here. So should I rename it to `@operator.repeat`?

Reply:
> define a base parser query definition

Yeah this would certainly make sense, the different queries are diverging already quite a bit (increasingly unfortunately).

This should probably be done in in the main tree-sitter repo I think, maybe start a RFC/discussion there. AFAIK maintainers of either editors are very open for such a change.

> So should I rename it to @operator.repeat?

Yeah I think it may be better to group things together and specify them hierarchically (as it's still an operator isn't it?). I'm not sure about nvim-treesitter, but helix just uses the highlight color for `@operator` if no `@operator.repeat` is defined (which isn't currently AFAIK).